classes/autoptimizeConfig.php

```diff
@@ -212,8 +217,45 @@ public function show() {
 
 </form>
 </div>
-<style>.autoptimize_banner ul li {font-size:medium;text-align:center;} .unslider-arrow {left:unset;}</style>
-<div class="autoptimize_banner">
+<style>
+.autoptimize_banner {
```

Comment:
can you remove these changes from the "toolbar"-pull request? I prefer to have one clean merge with only the great toolbar cache clean stuff. other improvements can be put in later, seperate pull requests :)